manim/mobject/three_dimensions.py

```diff
@@ -339,10 +329,8 @@ def construct(self):
         The direction of the apex.
     show_base : :class:`bool`
         Whether to show the base plane or not.
-    v_min : :class:`float`
-        The azimuthal angle to start at.
-    v_max : :class:`float`
-        The azimuthal angle to end at.
+    v_range : :class:`float`
```

Comment:
I would guess that `v_range` is no `float` anymore, no? More like a `Iterable[float]` maybe?

Reply:
Oh yes, i didn't notice that 😬 ill adjust that